standard/variables.md

```diff
@@ -1060,7 +1060,7 @@ For any variable, the ***ref-safe-context*** of that variable is the context whe
 
 There are three ref-safe-contexts:
 
-- ***declaration-block***: The ref-safe-context of a *variable_reference* to a local variable ([§9.2.9](variables.md#929-local-variables)) extends from its declaration to the end of the scope in which it is declared, including into any nested *embedded-statement*s in this range.
+- ***declaration-block***: The ref-safe-context of a *variable_reference* to a local variable ([§9.2.9](variables.md#929-local-variables)) is that local variable’s scope ([§13.6.2](statements.md#1362-local-variable-declarations)), including into any nested *embedded-statement*s in the scope.
```

Comment:
@KalleOlaviNiemitalo – "into" was added. The ", including into any nested *embedded-statement*s in the scope" is just referring redundantly (in both the original and this PR) to nesting (§7.7.1). Such redundant statements are made in various places where they are (subjectively!) deemed to be helpful. This PR keeps that redundancy simply because it was previously deemed helpful by somebody. Let’s see if others think it should now go.

Reply:
Small word changes: “any” removed as suggested by @KalleOlaviNiemitalo , “the scope” now “that scope”